lib/util/sdp.js

```diff
@@ -13,7 +38,7 @@ var chromeSdpFormat;
  * Get Chrome's default SDP format.
  * @returns {'planb'|'unified'}
  */
-function getChromeSdpFormat() {
```

Comment:
same comment as last one - I think splitting into two functions would make it more readable.